src/System.Private.CoreLib/shared/System/Type.cs

```diff
@@ -18,18 +19,18 @@ public abstract partial class Type : MemberInfo, IReflect
 
         public new Type GetType() => base.GetType();
 
-        public abstract string Namespace { get; }
-        public abstract string AssemblyQualifiedName { get; }
-        public abstract string FullName { get; }
+        public abstract string? Namespace { get; }
+        public abstract string? AssemblyQualifiedName { get; }
+        public abstract string? FullName { get; }
```

Comment:
Where do we return null from FullName and AssemblyQualifiedName?

Reply:
In SignatureType class
https://github.com/dotnet/coreclr/pull/23489/files#diff-ec5f1fed3ef1dd139eeb76c81dbd3b3eR73

Reply:
Could we change that to just return string.Empty instead and then have these be non-nullable?  I think developers would be surprised if these returned null, and I'm not even sure if we ever hand out SignatureType instances outside of corelib.

@jkotas, do you know if this would cause any problems?

Reply:
It is document to return null: https://docs.microsoft.com/en-us/dotnet/api/system.type.assemblyqualifiedname?view=netcore-3.0

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.type.fullname?view=netcore-3.0

Reply:
Huh, you're right, nevermind.